backend/app/api/endpoints/devices.py

```
@@ -438,3 +438,100 @@ async def execute_device_command(
+            user_id=current_user.id,
+            device_id=device_id,
+            project_id=0,
+            session_type="terminal",
+            path=DEFAULT_DEVICE_SESSION_PATH,
+            create_if_missing=True,
+        )
+    except DeviceSessionError as exc:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=str(exc),
+        )
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)